components/browse/ContentCardSection.vue

```diff
@@ -20,12 +20,25 @@
         deck
         data-qa="section group"
       >
-        <BrowseContentCard
-          v-for="card in cards"
-          :key="card.sys.id"
-          :fields="card.fields"
-          :card-type="card.sys.contentType ? card.sys.contentType.sys.id : ''"
-        />
+        <template v-if="!isPeopleSection">
+          <BrowseContentCard
+            v-for="card in cards"
+            :key="card.sys.id"
+            :fields="card.fields"
+            :card-type="card.sys.contentType ? card.sys.contentType.sys.id : ''"
+          />
+        </template>
+        <template v-else>
+          <ContentCard
+            v-for="card in cards"
+            :key="card.sys.id"
+            :title="card.fields.name"
+            :url="card.fields.identifier"
```

Comment:
The cards here link to the identifier itself, for example, http://data.europeana.eu/agent/base/147134. It does a couple of redirects and you finally end up on the production entity. 

Could you use a correct router link here? So it performs well and so you stay on the same domain.